### PR TITLE
feat: unsupported platform error

### PR DESCRIPTION
**Pull request by scovertb**
Produce compilation warning and log error when building/running on an unsupported platform

### CODE REVIEW

File: Assets/Plugins/Source/EOSManager.cs [hunk 1]
```diff
@@ -419,6 +419,11 @@ private void InitializeOverlay(IEOSCoroutineOwner coroutineOwner)
             // those cases, this code will early out.
             public void Init(IEOSCoroutineOwner coroutineOwner)
             {
+#if !UNITY_EDITOR && !(UNITY_STANDALONE_WIN)
```
Comment (pew-dahs) [inline Assets/Plugins/Source/EOSManager.cs:422]: nit: Is the extra parenthesis needed here?

**Review by pew-dahs — APPROVED** [review]
Looks good to me.

File: Assets/Plugins/Source/EOSManager.cs (hunk 1 above)
Comment (scovertb) [reply Assets/Plugins/Source/EOSManager.cs:422]: That's there because the parenthesized section will expand with more supported platforms.

**Comment by andrew-hirata-playeveryware** [comment]
I did a cherry-pick of this on to development instead, so we don't get unneeded commits from stable into development

**Pull request closed without merging**